api/apps/articles/test/model/index.coffee

```diff
@@ -682,6 +689,22 @@ describe 'Article', ->
         article.keywords.join(',').should.equal 'cool,art,Pablo Picasso,Pablo Picasso,Armory Show 2013,Gagosian Gallery,kana'
         done()
 
+    it 'indexes the article in elasticsearch on save', (done) ->
+      Article.save {
+        author_id: '5086df098523e60002000018'
+        title: 'foo article'
+        published: true
+      }, 'foo', (err, article) ->
+        return done err if err
+        sleep.sleep 1
```

Comment:
😮  interesting! Makes sense for the sake of integration testing. However, maybe we could instead stub `search.client` and avoid the need to request to ES in tests? Happy to save that for another PR, but quickly to note I think what you want is to use `setTimeout` instead here. That sleep module seems like a kinda strange debugging module that purposely blocks, whereas Node is designed to intentionally use non-blocking callback based APIs.

So this would look like...

````coffeescript
Article.save {
 ...
}, 'foo', (err, article) ->
  return done err if err
  setTimeout(=>
    search.client.search(...)
  , 1000)
````